skbio/sequence/_iupac_sequence.py

```diff
@@ -19,6 +19,13 @@
 from ._sequence import Sequence
 
 
+def _invert_case(array):
+    # ASCII is built such that the difference between uppercase and
+    # lowercase is the 6th bit.
+    array ^= 32
```

Comment:
Function gets my vote. The twiddling is opaque enough that I think it would require a comment to be duplicated everywhere it's used.